cloudevents/extensions/data-classification.md

```diff
@@ -0,0 +1,87 @@
+# Data Classification Extension
+
+CloudEvents may contain payload which is subjected to data protection
+regulations like GDPR or HIPAA. For intermediaries and consumers knowing how
+event payload is classified, which data protection regulation applies and how
+payload is categorized, enables compliant processing of an event.
+
+This extension defines attributes to describe to
+[consumers](../spec.md#consumer) or [intermediaries](../spec.md#intermediary)
+how an event and its payload is classified, category of the payload and any
+applicable data protection regulations.
+
+These attributes are intended for classification on an event and payload level
+and not on `data` field level. Classification on field level is best defined in
+the schema specified via the `dataschema` attribute.
+
+## Notational Conventions
+
+As with the main [CloudEvents specification](../spec.md), the key words "MUST",
+"MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT",
+"RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as
+described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+However, the scope of these key words is limited to when this extension is used.
+For example, an attribute being marked as "REQUIRED" does not mean it needs to
+be in all CloudEvents, rather it needs to be included only when this extension
+is being used.
+
+## Attributes
+
+### dataclassification
+
+- Type: `String`
+- Description: Data classification level for the event payload within the
+  context of a `dataregulation`. Typical labels are: `public`, `internal`,
+  `confidential`, `restricted`.
+- Constraints:
+  - REQUIRED
+  - SHOULD be applicable to data protection regulation.
```

Comment:
Can you elaborate on what this "SHOULD" means? What does someone need to do (from a coding perspective) to adhere to this "SHOULD"?

Reply:
The SHOULD statement is merely meant as an indication towards _event producers_ that the data classification label should have its origin within the applicable data-regulation. But maybe this is stating the obvious and from a coding perspective not relevant.  Being already stated in the _description_, it does not add value. I will remove it